README.md

```diff
@@ -1 +1,102 @@
-# onboarding
\ No newline at end of file
+# Onboarding
+
+## How to request your Demo LEAR Credential
+To begin the process of obtaining your Demo LEAR Credential, please send an email to [ruben.modamio@in2.es ](mailto:ruben.modamio@in2.es) with the required information listed below. This will help us create your credential efficiently. Note that the Mandator data can be extracted from your organization's EIDAS digital certificate.
+
+### Required Attributes:
++ DID Web name: The ID format should be did:web:<organizationDND> (e.g., did:web:in2.es)
+
+### Mandator Attributes:
++ OrganizationIdentifier: (e.g., VATES-B60645900)
```

Comment:
Is is better to ask for the VAT (Also called Tax Identification Number) of the organisation. And then we can create the `organizationIdentifier`.

Most people do not know the meaning of the `organizationIdentifier`.

Reply:
changed